packages/nimble-components/src/table-column/enum-base/types.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Input data types for mapping-based columns (i.e. enum text and icon)
+ * @public
+ */
+export const MappingKeyType = {
```

Comment:
nit: let's add a test for this type that matches the pattern of our other enums.

Reply:
Added